helm/prometheus-rules/templates/platform/atlas/alerting-rules/loki.rules.yml

```diff
@@ -195,3 +195,31 @@ spec:
         severity: page
         team: atlas
         topic: observability
+    - alert: LokiObjectStorageLowRate
+      annotations:
+        dashboard: loki-operational/loki-operational
+        description: '{{`Loki object storage write rate is down.`}}'
+        opsrecipe: loki/
+      expr: |
+        irate(loki_rate_store_stream_rate_bytes_count[5m]) == 0
```

Comment:
Why do we need the second part of the expression?

Reply:
Oh, good thinking. You might do this for loki canary because I think don't have that for them